Source/ACE.Database/WorldDatabase.cs

```diff
@@ -39,7 +39,10 @@ private enum WorldPreparedStatement
             GetAceObjectPropertiesPosition = 24,
             GetAceObjectPropertiesSpell = 25,
             GetAceObjectGeneratorLinks = 26,
-            GetMaxId = 27
+            GetMaxId = 27,
+            GetAceObjectPropertiesAttributes,
```

Comment:
I don't like the inconsistency here.  Either remove the = # from all of them, or put the = # in your enum values.